logic/randomize.py

```diff
@@ -631,3 +632,58 @@ def randomize_dungeons_trials_starting_entrances(self):
         assert self.randomized_start_entrance["layer"] is not None
         assert self.randomized_start_entrance["entrance"] is not None
         assert self.randomized_start_entrance["day-night"] is not None
+
+        # Starting bird statue rando
+
+        bsr = self.options["random-start-statues"]
+
+        possible_bird_statues = [
+            (entrance, values)
+            for entrance, values in self.areas.map_entrances.items()
+            if values.get("subtype", False)
+            and values["subtype"] == "bird-statue-entrance"
```

Comment:
```suggestion
            if values.get("subtype") == "bird-statue-entrance"
```
comparing None is fine